manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -0,0 +1,104 @@
+{{- if .Values.arc.isMonitoringEnabled }}
+{{- if .Values.linux.enabled}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sscdpa.arc.fullname" . }}-metrics-monitoring
+  namespace: {{ .Release.Namespace }}
+{{ include "sscdpa.arc.labels" . | indent 2 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "sscdpa.arc.fullname" . }}
+  template:
+    metadata:
+{{ include "sscdpa.arc.labels" . | indent 6 }}
+    spec:
+      serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
+      containers:
+        - name: prom-mdm-converter
+          image: "{{ .Values.arc.metrics.promMdmConverter.image.repository }}:{{ .Values.arc.metrics.promMdmConverter.image.tag }}"
+          imagePullPolicy: {{ .Values.arc.metrics.promMdmConverter.image.pullPolicy }}
+          resources:
+{{ toYaml .Values.linux.resources | indent 12 }}
```

Comment:
We don't want to reuse the provider pod resources here. I would recommend having separate resources for this container and setting appropriate values for it.